src/popup/components/TransferReview.vue

```diff
@@ -279,7 +309,15 @@ export default defineComponent({
       }
     }
 
-    async function submit() {
+    // TODO
+    function proposeMultisigTransaction() {
+      root.$store.dispatch('modals/open', {
```

Comment:
if my pr will be merged before that yours,  then please use useDIspatch

Reply:
Sure. But it's not merged :)